src/store/journal/journal.reducer.ts

```diff
@@ -0,0 +1,38 @@
+import { createFeatureSelector } from '@ngrx/store';
+
+import * as journalActions from './journal.actions';
+import { JournalModel } from './journal.model';
+import { StoreModel } from '../store.model';
+
+export const initialState: StoreModel = {
+  journal: {
+    isLoading: false,
+    testSlot: [],
+  }
+};
+
+export function journalReducer(state = initialState.journal, action: journalActions.Types) {
```

Comment:
also set the return type to be `JournalModel`
`export function journalReducer(state = initialState, action: journalActions.Types): JournalModel {`

Reply:
Added the return type.